js/analyzer.js

```diff
@@ -159,9 +164,9 @@ Analyzer.prototype.keywordCount = function() {
  * checks if keywords appear in subheaders of stored cleanTextSomeTags text.
  * @returns resultObject
  */
-Analyzer.prototype.subHeadings = function() {
+YoastSEO_Analyzer.prototype.subHeadings = function() {
     var result = [ {test: "subHeadings", result: {count: 0, matches: 0 } } ];
-    var matches = this.preProcessor.__store.cleanTextSomeTags.match( /<h([1-6])(?:[^>]+)?>(.*?)<\/h\1>/g );
+    var matches = this.YoastSEO_preProcessor.__store.cleanTextSomeTags.match( /<h([1-6])(?:[^>]+)?>(.*?)<\/h\1>/ig );
```

Comment:
Perhaps it's wise to add inline comments to explain that this part checks all text within the tags <h1> through <h6>. Also explain the regex.